domains.txt

```diff
@@ -200,14 +230,17 @@ email.welovesalt.com
 empiric.com
 empiricsolutions.co.uk
 emponics.co.uk
+emr.co.uk
```

Comment:
EMR is a specialist marketing recruitment agency.
We cover everything from core marketing through to digital, communications and investor relations.

We’ve always believed that marketers have the power to shape the future of businesses and that many of today’s marketers will be the future CEOs, MDs and entrepreneurs of the digital age.